lyluatex.lua

```diff
@@ -804,15 +814,39 @@ end
 function ly.file(input_file, options)
     --[[ Here, we only take in account global option includepaths,
     as it really doesn't mean anything as a local option. ]]
-    input_file = locate(input_file, Score.includepaths)
+    local filename = input_file
+    input_file = locate(input_file, Score.includepaths, '.ly')
     options = ly.set_local_options(options)
-    if not input_file then err("File %s.ly doesn't exist.", input_file) end
+    if not input_file then err("File %s doesn't exist.", filename) end
     local i = io.open(input_file, 'r')
     ly.score = Score:new(i:read('*a'), options, input_file)
     i:close()
 end
 
 
+function ly.file_musicxml(input_file, options)
+    --[[ Here, we only take in account global option includepaths,
+    as it really doesn't mean anything as a local option. ]]
+    local filename = input_file
+    input_file = locate(input_file, Score.includepaths, '.xml')
```

Comment:
Hm, MusicXML files have a number of typical extensions, including `.mxl` and `.musicxml`.
I'm not sure how to deal with that. The "right" way would be to make this an option, but I have the impression this is overkill.

Reply:
Those extensions (`.ly`, `.xml`) are only a fallback in case the user didn't explicitly specify the extension. so if someone types:
```tex
\lilypondfile{music.ily}
```
it will work; but if he types:
```tex
\lilypondfile{music}
```
it will look not only for `music`, but also for `music.ly`.
So I only took in account the default extension for this, as the user can type other extensions explicitly.